dietstory-routing/game_asset_manager/views.py

```diff
@@ -189,44 +193,100 @@ def post(self, request, *args, **kwargs):
         """
         game_version_form = SubmitGameVersionForm(request.data)
 
-        if game_version_form.is_valid():
+        if not game_version_form.is_valid():
+            return JsonResponse(
+                {'message': 'Invalid input parameters'},
+                status=status.HTTP_400_BAD_REQUEST)
 
-            try:
-                major_ver, minor_ver = game_version_form.get_version_values();
-                live_by = game_version_form.cleaned_data.get('live_by')
+        major_ver, minor_ver = game_version_form.get_version_values();
+        live_by = game_version_form.cleaned_data.get('live_by')
 
-                game_version = GameVersions(major_ver=major_ver, minor_ver=minor_ver, live_by=live_by)
+        # Check if Game Version Already Exists
+        try:
+            exists = GameVersions.objects.get(major_ver=major_ver, minor_ver=minor_ver)
+        except GameVersions.DoesNotExist:
+            exists = None
+        if exists:
+            return JsonResponse(
+                {'message': 'Version already exists'},
```

Comment:
Did you add the response 304 to swagger? Other than that, code looks in good shape.

Reply:
Swagger response for 304 has been added.